Libraries/LibJS/Tests/Array.prototype.fill.js

```diff
@@ -0,0 +1,34 @@
+load("test-common.js");
+
+Array.prototype.equals = function(other) {
```

Comment:
Please don't do this - write a helper function in `test-common.js` instead :)

Also strict equality is preferred whenever possible - this function would declare `[1, 2, 3]` and `["1", "2", "3"]` equal!